compiler/damlc/lib/DA/Cli/Damlc/DependencyDb.hs

```diff
@@ -265,6 +296,15 @@ queryDalfs markersM dir = do
                          forM markers $ \marker -> doesFileExist $ takeDirectory fp </> marker)
                     dalfs
 
+queryPkgIds :: Maybe [FilePath] -> FilePath -> IO [LF.PackageId]
+queryPkgIds markersM dir = do
+    fps <- queryDalfs markersM dir
+    pure
+        [ LF.PackageId $ T.pack pkgId
+        | fp <- fps
+        , _dalf:pkgId:_rest <- [reverse $ splitDirectories fp]
```

Comment:
I don’t understand the `reverse`. I thought the pkg id is the second directory from the top not from bottom? When do we actually have more than 3?

Reply:
yes, it should be the second from the top, but it seems I get them in the wrong order from `splitDirectories`. Let me check again.

Reply:
ok, definitely a friday evening decision. I removed the reverse.